daemon/Cargo.toml

```diff
@@ -44,3 +44,22 @@ rust-ini = "0.21.1"
 assert_matches = "1.5"
 serial_test = "3.1.1"
 tracing-test = "0.2.4"
+
+[profile.release]
+# Stripping symbols (not debuginfo) reduces binary size by ~1.5x.
+strip = true
+opt-level = 3
+
+# 2-3x slower to compile, but produces a ~1.2x smaller binary.
+# TODO: benchmark this!
```

Comment:
I'm not sure if we should leave the TODO in... What would you want to benchmark there? I think for this use-case the more handwavy "it's probably LTO" could be enough :-)
```suggestion
```

Reply:
Yes! I was mostly putting that in there in case someone wondered why the code was there if it's not being used! Didn't want to leave little bits of nonfunctional code around, but I like your comment quite a bit and should give any reader the clues they need to see how to use it :)